c_src/membrane_element_sdl/player.c

```diff
@@ -7,92 +6,98 @@
 
 #include "player.h"
 
-int create(int width, int height, State *state) {
-  SDL_Window *window = NULL;
-  SDL_Renderer *renderer = NULL;
-  SDL_Texture *texture = NULL;
+UNIFEX_TERM create(UnifexEnv *env, int width, int height) {
+  char error[2048] = {0};
```

Comment:
I am not sure if this is a standard but in some repos I have seen that instead of array for error string there is a flag like `errno` which is set in case of error and function `get_error_str` or sth similar which is called when string representation of the error is needed. We would also need some macros for errors like `#define SDL_FRAME_ERR some_int`. It may be an overkill in our use case so I don't know which approach is better.

Reply:
I believe errno is useful when creating API for someone else to use, not when you just want to raise an exception